src/modals/SendModal/BuildTransferTx.tsx

```diff
@@ -79,3 +93,13 @@ const BuildTransferTx = ({ data, onSubmit, onCancel }: BuildTransferTxProps) =>
 }
 
 export default BuildTransferTx
+
+function useStateWithError<T>(initialValue: T) {
```

Comment:
Is it possible to reuse the one in `utils.txs`?

Reply:
Or is it moved here from `utils.txs`?

Reply:
It is moved here, since it's not used anywhere else